composables/useErc725.ts

```diff
@@ -1,33 +1,38 @@
 import { ERC725, ERC725JSONSchema } from '@erc725/erc725.js'
 import LSP3UniversalProfileMetadata from '@erc725/erc725.js/schemas/LSP3UniversalProfileMetadata.json'
+import LSP4DigitalAsset from '@erc725/erc725.js/schemas/LSP4DigitalAsset.json'
 import { LSP3Profile } from '@lukso/lsp-factory.js'
 import Web3 from 'web3'
+import { LSP4DigitalAssetJSON } from '@lukso/lsp-factory.js/build/main/src/lib/interfaces/lsp4-digital-asset'
+import { URLDataWithHash } from '@erc725/erc725.js/build/main/src/types'
+import { ERC725YDataKeys } from '@lukso/lsp-smart-contracts'
 
+import { SupportedAssets, TokenIdType } from '@/types/assets'
 import { getImageUrlBySize } from '@/utils/getProfileImages'
+import LSP8IdentifiableDigitalAsset from '@/shared/schemas/LSP8IdentifiableDigitalAsset.json'
+import { PROVIDERS } from '@/types/enums'
+import { getDataABI } from '@/shared/abis/getDataABI'
 
 export interface LSP3ProfileJSON {
   LSP3Profile: LSP3Profile
 }
 
-const getInstance = (address: string) => {
-  const network = useAppStore().getNetwork(useAppStore().selectedNetwork)
-  const provider = new Web3.providers.HttpProvider(network.rpcHttp)
+const getInstance = (address: string, schema: ERC725JSONSchema[]) => {
+  const { currentNetwork } = useAppStore()
   const config = {
-    ipfsGateway: network.ipfsUrl,
+    ipfsGateway: currentNetwork.ipfsUrl,
```

Comment:
why is IPFS related to the network? It should not?

We use the same IPFS for any L14, L16, testnet, mainnet etc.

Reply:
I don't think it is needed to put the ipfsUrl in the network settings. It is not related.

Reply:
Alright I will change this. I was thinking that we might want to use different storage per network but I guess I was wrong.